build/__snapshots__/imported-features.json

```diff
@@ -0,0 +1,193 @@
+[
+  "action-pr-link",
```

Comment:
This is great. We now have a static JSON file with a list of features.

This could enable things like #4493 😜 

It can potentially also be transformed to a raw-text list